eos/models/standard-model.cc

```diff
@@ -814,22 +898,16 @@ namespace implementation
 
         // only unprimed WCs are non-zero in the SM
         // leading order in alpha_s
-        const std::array<double, 10u> lo_unprimed = {
-            -1.0 / 9.0, -2.0 / 3.0, +1.0 / 36.0, +1.0 / 6.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0
-        };
+        const std::array<double, 10u> lo_unprimed  = { -1.0 / 9.0, -2.0 / 3.0, +1.0 / 36.0, +1.0 / 6.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0 };
         // next-to-leading order in alpha_s
-        const double L = 2.0 * std::log(_mu_0__sbcu / _m_W__sbcu);
+        const double                  L            = 2.0 * std::log(_mu_0__sbcu / _m_W__sbcu);
```

Comment:
hmm --"

Reply:
That's a Github display problem, see the screenshot:
![image](https://github.com/user-attachments/assets/573a38e6-b304-448f-85ba-d727f5c9149a)

Reply:
It is perfectly aligned:

![image](https://github.com/user-attachments/assets/51fad1a1-3107-4a2a-b27b-596da35c7922)

Reply:
Sorry, this is ``dbcu``, but ``sbcu`` looks the same.

Reply:
Yes, it is ugly but I understand